### PR TITLE
Start to port venues form to new styling

### PR DESCRIPTION
**Pull request by jayzes**
- [ ] Venue new/edit form
- [ ] Venue availability form

### CODE REVIEW

File: app/views/components/_venue_card.html.slim [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  .VenueCard
+    .VenueCard-link
+      = link_to edit_venue_path(venue)  do
+        = render partial: 'icons/pencil'
+    h2.VenueCard-title #{venue.name}
+    .VenueCard-address
+      . #{venue.address}
+      . #{venue.city}, #{venue.state}
+    p.VenueCard-description #{venue.description}
+    .ContentCard-action
```
Comment (scpaye506) [inline app/views/components/_venue_card.html.slim]: is this `.ContentCard-action` being used?

File: app/assets/stylesheets/redesign/components/venue_edit.sass [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.VenueEdit-availabilities
+  margin-top: 25px
+
+.VenueEdit-submit
+  +white-button(315px, 56px)
+  margin: 40px auto 0
+  width: 315px
+
+table
```
Comment (scpaye506) [inline app/assets/stylesheets/redesign/components/venue_edit.sass]: let's add classes to these table elements or nest them within some sort of class so these styles don't leak into global styles. Incase there are ever tables somewhere else

File: app/views/venues/edit.html.slim [hunk 3]
```diff
@@ -1,19 +1,16 @@
-= render partial: 'dashboards/nav', locals: { item: :venue }
+- content_for(:fullscreen_takeover, true)
```
Comment (scpaye506) [inline app/views/venues/edit.html.slim]: it's probably worth putting all of this into the venue form component and then being able to reuse the entire form in both the new and edit files (even if we aren't using new for now). You can follow the pattern setup in articles/articles_form and articles/new and articles/edit files

File: app/controllers/venues_controller.rb [hunk 4]
```diff
@@ -4,11 +4,6 @@ class VenuesController < ApplicationController
   before_action :set_venue, only: %i[show edit update destroy]
```
Comment (scpaye506) [inline app/controllers/venues_controller.rb]: add `skip_before_action :store_location, only: %i[new edit]` above this `before_action` forgot to mention this, but it helps with closing the fullscreen takeover screens

File: app/views/components/_venue_card.html.slim (hunk 1 above)
Comment (andylushman) [reply app/views/components/_venue_card.html.slim]: just removed it.

File: app/controllers/venues_controller.rb (hunk 4 above)
Comment (andylushman) [reply app/controllers/venues_controller.rb]: Sounds good. Done.

File: app/assets/stylesheets/redesign/components/venue_edit.sass (hunk 2 above)
Comment (andylushman) [reply app/assets/stylesheets/redesign/components/venue_edit.sass]: Good call! Just created `VenueEdit-table` and nested the rest over it.

**Pull request merged**